server/implementation/src/main/java/io/smallrye/graphql/execution/datafetcher/AbstractDataFetcher.java

```diff
@@ -49,8 +49,11 @@ public AbstractDataFetcher(Operation operation, Config config) {
     public T get(final DataFetchingEnvironment dfe) throws Exception {
         // update the context
         GraphQLContext graphQLContext = dfe.getContext();
-        SmallRyeContext context = ((SmallRyeContext) graphQLContext.get("context")).withDataFromFetcher(dfe, operation);
```

Comment:
Yea, it can't, this was as part of my testing. But, I am keeping this as there is an upcoming feature to disable context that I am going to work on soon